nodejs/awsx/autoscaling/autoscaling.ts

```diff
@@ -85,7 +83,7 @@ export class AutoScalingGroup extends pulumi.ComponentResource {
 
         // Now go and actually find the group created by cloudformation.  The id for the group will
         // be stored in `stack.outputs.Instances`.
-        this.group = aws.autoscaling.Group.get(name, this.stack.outputs["Instances"], undefined, { parent: this });
+        this.group = aws.autoscaling.Group.get(name, <any>this.stack.outputs["Instances"], undefined, { parent: this });
```

Comment:
not certain what's going on here.  TS now considers this to be an `Output<unknown>` which is not compatible with the `Output<string>` we need.  I think this is a change in the latest version of TS.  It's likely a more correct typing indicating: "you don't know that 'instances' actually exists.  we just workaround this for now with a trival `any` case.